DependencyInjection/CmfResourceExtension.php

```diff
@@ -34,64 +35,120 @@ public function load(array $configs, ContainerBuilder $container)
 
         $config = $processor->processConfiguration($configuration, $configs);
 
-        $this->loadRepositories($config['repository'], $container);
+        $this->loadRepositories($config['repositories'], $container);
     }
 
-    private function loadRepositories($config, $container)
+    private function loadRepositories(array $repositories, ContainerBuilder $container)
     {
-        foreach (array(
+        $methods = array(
             'composite' => 'createCompositeRepository',
             'doctrine_phpcr' => 'createDoctrinePhpcrRepository',
             'doctrine_phpcr_odm' => 'createDoctrinePhpcrOdmRepository',
             'filesystem' => 'createFilesystemRepository',
```

Comment:
I think we should ultimately make the DI extension repository-type agnostic, but we can do that later.